x-pack/plugins/stack_alerts/server/rule_types/es_query/rule_type.ts

```diff
@@ -134,6 +136,14 @@ export function getRuleType(
     }
   );
 
+  const alerts: IRuleTypeAlerts = {
+    context: 'observability.esquery',
+    mappings: { fieldMap: { [ALERT_CONTEXT]: { type: 'object', array: false, required: false } } },
+    useEcs: false,
+    useLegacyAlerts: false,
+    shouldWrite: true,
+  };
+
```

Comment:
I used `observability` prefix to get the alerts in Observability Alerts table but didn't work. 
Probably we need a different index name here.

Mappings should be aligned with Es Query rule type fields as well.